azure/pipelines/node-npm/install-volta.yml

```diff
@@ -0,0 +1,35 @@
+# Install Volta Node version manager in your pipeline
+# https://wiki.delaware.pro/pages/viewpage.action?pageId=314778333
+
+steps:
+- bash: |
+   echo "Downloading Volta..."
+   curl https://get.volta.sh | bash -s -- --skip-setup
+
+   echo "Preparing environment variables..."
+   export VOLTA_HOME="$HOME/.volta"
+   export PATH="$VOLTA_HOME/bin:$PATH"
+   
+   echo "##vso[task.setvariable variable=VOLTA_HOME]$VOLTA_HOME"
+   echo "##vso[task.setvariable variable=PATH]$PATH"
+   
+   echo "Starting installation..."
+   volta setup --verbose
+  condition: or(eq( variables['Agent.OS'], 'Linux' ), eq( variables['Agent.OS'], 'Darwin' ))
+  displayName: Install Volta for Unix
+- powershell: |
+   $voltaVersion = "1.1.0"
```

Comment:
Shouldn't this be a parameter (with a default value)? How about choosing a specific version when on linux?

Reply:
Good point about a specific version for Linux. We'll have to expand this and follow the procedure over at Volta
https://docs.volta.sh/advanced/installers

Reply:
Parameters don't get picked up when using a template it seems. Do you want to see this parameter as an option before you click "Run" or pass it to the template as a parameter?

Reply:
Just add a parameter here, the pipeline that uses this template can choose what to do with it